CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## v1.20.0.0
+
+* change `workflows/main.tf` to reference `v1.20.0` terraform module
+
 ## v1.19.0.0
 
-* Single change to pull the correct cumulus terraform version
```

Comment:
Double checking that this change was intentional? Just retroactively updating the docs to confirm to the new change format?

Reply:
+1 and I'm good with the rest of it. You may merge when ready! :)